rekcurd_dashboard/apis/api_evaluation.py

```diff
@@ -245,9 +248,25 @@ def post(self, project_id: int, application_id: str):
         return response_body
 
 
-@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluation_results/<int:eval_result_id>')
+@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluation_results')
 class ApiEvaluationResults(Resource):
+    def get(self, project_id: int, application_id: str):
+        """get all evaluation results and belonging models and evaluation data"""
+        rows = db.session.query(EvaluationModel, EvaluationResultModel, ModelModel)\
+            .filter(EvaluationModel.application_id == application_id,
+                    EvaluationResultModel.evaluation_id == EvaluationModel.evaluation_id,
+                    EvaluationResultModel.model_id == ModelModel.model_id).all()
+        results = []
+        for row in rows:
+            result = row.EvaluationResultModel.serialize
+            result['model'] = row.ModelModel.serialize
+            result['evaluation'] = row.EvaluationModel.serialize
+            results.append(result)
+        return results
+
 
+@evaluation_api_namespace.route('/projects/<int:project_id>/applications/<application_id>/evaluation_results/<int:eval_result_id>')
+class ApiEvaluationResultId(Resource):
     def get(self, project_id: int, application_id: str, eval_result_id: int):
```

Comment:
ditto

Reply:
fixed in https://github.com/rekcurd/dashboard/pull/105/commits/e0a21c7d65247b96070541c49a52202fbef180cc